LayoutTests/platform/gtk/TestExpectations

```diff
@@ -2523,3 +2523,6 @@ imported/w3c/web-platform-tests/css/css-contain/content-visibility/content-visib
 
 animations/font-variations/font-stretch.html [ Crash Pass ]
 fast/mediastream/mediastreamtrack-video-frameRate-clone-decreasing.html [ Crash Pass ]
+
+webkit.org/b/260625 media/media-can-play-case-sensitive-flac.html [ Failure ]
+webkit.org/b/260625 media/media-can-play-type-case-sensitive.html [ Failure ]
```

Comment:
Can you move this to platform/glib/TestExpectations please? That's the shared file for wpe and gtk